thoth/storages/graph/models.py

```diff
@@ -68,6 +68,7 @@ class PythonPackageVersion(Base, BaseExtension):
     python_package_metadata_id = Column(
         Integer, ForeignKey("python_package_metadata.id", ondelete="CASCADE"), nullable=True
     )
+    is_missing = Column(Boolean, nullable=False, default=False)
```

Comment:
Please provide also the corresponding alembic migration :) https://github.com/thoth-station/storages/blob/master/README.rst#generating-migrations-and-schema-adjustment-in-deployment